.github/workflows/horus.yml

```diff
@@ -19,11 +19,13 @@ jobs:
     - name: Get the list of packages
       run: |
         cp jammy/packages_to_import /tmp/patched-packages
-    - name: Install Dependencies
+    - name: Install Dependencies and enable Sources
       run: |
         sed -i 's/^\(Types: deb\)$/\1 deb-src/g' /etc/apt/sources.list.d/ubuntu.sources
         apt update
-        apt install -y git python3-launchpadlib python3-apt python3-github
+        apt install -y git python3-launchpadlib python3-apt python3-github python3-pip
+        pip install --break-system-packages GitPython
```

Comment:
Is there any reason why you don't use `python3-git` from apt?

Reply:
I just couldn't find how it is named. apt should be the correct approach here.